docs/cyber.md

```diff
@@ -0,0 +1,145 @@
+<<<<<<< .merge_file_a06828
```

Comment:
this has alot of git artifacts in it you should remove these and ensurethe new docs point to the latest classes

Reply:
fixed and uploaded